### PR TITLE
chore: add drop-checker page

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Related: https://github.com/kodadot/ops-internal/issues/1768

Demo: https://deploy-preview-10578--koda-canary.netlify.app/drop-checker/ahp/158

Screenshot 2024-07-11 at 16 59 28

### CODE REVIEW

**Comment by vikiival** [comment]
Scr?

**Review by Jarsen136 — APPROVED** [review]
looks cool

**Review by hassnian — COMMENTED** [review]
nice!

just a thought,  what if we added ❌ or ✅ if the onchain and database data was `=== `  similar to what we do in the code-cheker

CleanShot 2024-07-13 at 12 26 23@2x

small darkmoda stuff, otherwise LGTM

CleanShot 2024-07-13 at 12 17 22@2x

can't see the text here 
CleanShot 2024-07-13 at 12 14 36@2x

**Comment by exezbcz** [comment]
looks good, where can i access the page?

**Comment by vikiival** [comment]
> looks good, where can i access the page?

> Demo: https://deploy-preview-10578--koda-canary.netlify.app/drop-checker/ahp/158

so `<url>/drop-checker/<prefix>/<id>`

**Comment by preschian** [comment]
> just a thought, what if we added ❌ or ✅ if the onchain and database data was `===` similar to what we do in the code-cheker

added 👍🏻



> small darkmoda stuff
> can't see the text here

fixed

File: pages/drop-checker/[chain]/[collection-id].vue [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+const collectionId = params['collectionid'].toString()
+
+const { chainSymbol, decimals } = useChain()
+
+// get data from database
+const data = await getDrops({
+  chain: [chain],
+  limit: 10,
+})
+const drops = data.filter((drop) => drop.collection === collectionId)[0]
```
Comment (Jarsen136) [inline pages/drop-checker/[chain]/[collection-id].vue:250]: It only gets information from the first 10 drops. Could we make it 100 or so?

**Review by Jarsen136 — APPROVED** [review]

**Review by hassnian — APPROVED** [review]

File: pages/drop-checker/[chain]/[collection-id].vue (hunk 1 above)
Comment (preschian) [reply pages/drop-checker/[chain]/[collection-id].vue:250]: oh, correct, let's use 100 for now
later on, we need to fetch drop by chain & collection id: https://github.com/kodadot/private-workers/issues/200

**Pull request merged**